src/plugins/OSDReferences.js

```diff
@@ -12,32 +9,3 @@ export const OSDReferences = {
     this.refs[windowId] = ref;
   },
 };
-
-/** */
-class OSDReferenceComponent extends React.Component {
```

Comment:
Is it the case that other plugins will not need to mount this anymore? Or that they should just use the provided add plugin? Will that work with two plugins needing this interface?

Reply:
Correct; applications that want an OSD reference don't need to do anything special, we just always stash a ref away into  `OSDReferences` (where plugins are getting their data from).